client/db/bolt/upgrades.go

```diff
@@ -269,6 +280,76 @@ func v5Upgrade(dbtx *bbolt.Tx) error {
 // 	})
 // }
 
+// v6Upgrade moves active matches from what will become the
+// archivedMatchesBucket to a new matchesBucket. This is done in order to make
+// searching active matches faster, as they do not need to be pulled out of all
+// matches any longer. This upgrade moves active matches as opposed to inactive
+// matches under the assumption that there are less active matches to move, and
+// so a smaller database transaction occurs.
+//
+// NOTE: Earlier taker cancel order matches may be MatchComplete AND have an
+// Address set because the msgjson.Match included the maker's/trade's address.
+// However, this upgrades does not patch the address field because MatchIsActive
+// instead keys off of InitSig to detect cancel matches, and this is a
+// potentially huge set of matches and bolt eats too much memory with ForEach.
+func v6Upgrade(dbtx *bbolt.Tx) error {
+	const oldVersion = 5
+
+	if err := ensureVersion(dbtx, oldVersion); err != nil {
+		return err
+	}
+
+	oldMatchesBucket := []byte("matches")
+	newActiveMatchesBucket := []byte("activeMatches")
```

Comment:
Not using `activeMatchesBucket` and `archivedMatchesBucket` because this is better semantics?

Reply:
Missed this comment, sorry.  Same consideration as other comment - a v7+ upgrade might change the package-level variables, but this v6 upgrade will always need to work with a specific key as it was in v5